packages/strapi-admin/admin/src/components/OnboardingVideo/styles.scss

```diff
@@ -0,0 +1,144 @@
+li.listItem {
+  display: block;
+  padding: 17px 15px;
+  cursor: pointer;
+  &:hover {
+    background-color: #f7f8f8;
+    .title {
+      color: #0e7de7;
+    }
+  }
+  .txtWrapper,
+  .thumbWrapper {
+    display: inline-block;
+    vertical-align: middle;
+  }
+  .thumbWrapper {
+    position: relative;
+    width: 55px;
+    height: 38px;
+    background-color: #d8d8d8;
+    border-radius: 2px;
+    overflow: hidden;
+    .overlay {
+      position: absolute;
+      top: 0;
+      left: 0;
+      z-index: 1;
+      width: 100%;
+      height: 100%;
+      background-color: rgba(#0E7DE7, .8);
+    }
+    img {
+      position: relative;
+      z-index: 0;
+      width: 100%;
+      height: 100%;
+    }
+    .play {
+      position: absolute;
+      top: 50%;
+      margin-top: -10px;
+      left: 50%;
+      margin-left: -10px;
+      width: 20px;
+      height: 20px;
+      background-color: #0e7de7;
+      border: 1px solid white;
+      text-align: center;
+      line-height: 20px;
+      border-radius: 50%;
+      z-index: 2;
+      &::before {
+        content: '\f04b';
+        display: inline-block;
+        vertical-align: top;
+        height: 100%;
+        font-family: 'FontAwesome';
+        color: white;
+        font-size: 10px;
+        margin-left: 3px;
+        line-height: 18px;
+      }
+    }
+  }
+  &.finished {
+    .title {
+      color: #919bae;
+    }
+    .thumbWrapper {
+      .overlay {
+        background-color: transparent;
+      }
+      img {
+        opacity: 0.6;
+      }
+      .play {
+        background-color: #5a9e06;
+        border-color: #5a9e06;
+        &::before {
+          content: '\f00c';
+          margin-left: 0;
+          font-size: 11px;
+          line-height: 20px;
+        }
+      }
+    }
+  }
+
+  .txtWrapper {
+    padding: 0 15px;
+    p {
+      font-size: 14px;
+      line-height: 24px;
+      font-family: Lato-SemiBold;
```

Comment:
Just asking... but when I see this file (https://github.com/strapi/strapi/blob/master/packages/strapi-admin/admin/src/styles/base/fonts.scss), we are using the `font-weight` property to determine how "bold" the font will be displayed.

I don't remember that we were doing things like that, otherwise, it means that we import fonts twice.

Reply:
Btw, it might explain why the fonts are not working well on Firefox and Safari...